### ISSUE TITLE
Incorrect Unirule mapping: GO:0022626 | cytosolic ribosome | IEA with UR000376628 | GO_REF:0000104

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#5572]
**UniProt UniRule (ID and label):**

GO:0022626 | cytosolic ribosome | IEA with UR000376628 | GO_REF:0000104

**Description of issue**
https://www.pombase.org/gene/SPBC3E7.10

initiator methionyl aminopeptidase isn't part_of ribosome

**Comment by pedruzzi** [comment]
You're correct, this GO term should not be annotated like this here. Initiator methionyl aminopeptidase associates with the cytosolic ribosome to process substrates cotranslationally, the correct annotation to the GO term would be "is_active_in" GO:0022626 | cytosolic ribosome. Unfortunately, we cannot control the qualifier via the rules, so it takes it's default "part_of", therefore I removed the GO term completely from the rule.
